middleware.go

```diff
@@ -23,7 +24,9 @@ const (
 // requests. The serverName parameter should describe the name of the
 // (virtual) server handling the request.
 func Middleware(serverName string, opts ...Option) func(next http.Handler) http.Handler {
-	cfg := config{}
+	cfg := config{
+		TraceSampledResponseKey: defaultTraceSampledResponseHeaderKey,
```

Comment:
I think it's better to set this in [here](https://github.com/riandyrn/otelchi/blob/master/config.go#L100-L104).

Reply:
done, PTAL